cg_hermes/constants/tags.py

```diff
@@ -0,0 +1,637 @@
+from enum import StrEnum
+from typing import Any
+
+
+class AlignmentTags(StrEnum):
+    BAM: str = "bam"
+    BAM_INDEX: str = "bam-index"
+    BAM_MT: str = "bam-mt"
+    CRAM: str = "cram"
+    CRAM_INDEX: str = "cram-index"
+
+    @property
+    def description(self):
```

Comment:
Another option would be to have a class for descriptions and have all the descriptions together regardless of the tag class type. Might be a bit cleaner and easier to read but feel free to disregard